bgpd/bgp_attr.h

```diff
@@ -250,6 +250,9 @@ struct attr {
 
 	/* rmap set table */
 	uint32_t rmap_table_id;
+
+	/* route map set mtu */
+	uint32_t mtu;
```

Comment:
uint16_t isn't sufficient for MTU sizes?

Reply:
we store it as 32 bit in zebra and the kernel takes a 32 bit value too!
I'll make the change a bit later

Reply:
why are you setting this in attribute?  If we ever support any of the mtu attributes being discussed in the ietf (which are individual drafts) this is going to be confusing. Perhaps  bgp_path_info_extra would be a better place to carry this...

Reply:
This is another one of those "there really isn't a good place to stuff this information" kinds of things, probably... Since BGP doesn't actually carry this anyplace, and its per route, though, I think extra is going to be better than attribute.

Reply:
I think we need to figure out where to put this data--should we do a meeting or something? I know we all love more meetings, so... @donaldsharp ?? @louberger ??